api/src/main/java/org/pentaho/metaverse/api/analyzer/kettle/ExternalResourceCache.java

```diff
@@ -45,9 +49,9 @@
 
   private static final Logger log = LoggerFactory.getLogger( ExternalResourceCache.class );
 
-  protected volatile Map<String, TransValues> transMap = new ConcurrentHashMap();
```

Comment:
Rationale for removing transMap and all related logic:
TransValues is really just a collection of transformations.
TransValues isn't used outside of the this class. So since no External class uses it, makes no sense to cache it.